tests/testsuite/cargo_information/pick_msrv_compatible_package_within_ws_and_use_msrv_from_root_manifest/mod.rs

```diff
@@ -0,0 +1,31 @@
+use cargo_test_macro::cargo_test;
+use cargo_test_support::{compare::assert_ui, curr_dir, Project};
+
+use super::{cargo_info, init_registry_without_token};
+
+#[cargo_test]
+fn case() {
+    init_registry_without_token();
+    cargo_test_support::registry::Package::new("my-package", "0.1.0").publish();
+    cargo_test_support::registry::Package::new("my-package", "0.2.0")
+        .rust_version("1.0.0")
+        .publish();
+    cargo_test_support::registry::Package::new("my-package", "0.2.1")
+        .rust_version("1.70.0")
```

Comment:
Should we be using outrageously high numbers?

Reply:
The reason I picked this version is I want to make sure we don't use global rustc from the local env. We should use the MSRV from crate2.